src/main/java/Company.java

```diff
@@ -0,0 +1,92 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Company {
+    private Employee employee;
+    private static final String IT_DEPARTMENT = "IT";
+    private static final String SUPPORT_DEPARTMENT = "Support";
+    private static final String MANAGEMENT_DEPARTMENT = "Management";
+
+    static Employee[] readFile(File file) throws FileNotFoundException, NumberFormatException {
+        Scanner scanner = new Scanner(file);
+        int lines = countLines(file);
+        Employee[] employees = new Employee[lines];
+        for (int i = 0; i < lines; i++) {
+            String line = scanner.nextLine();
+            String[] split = line.split(";");
+            String firstName = split[0];
+            String lastName = split[1];
+            String pesel = split[2];
+            String department = split[3];
+            double salary = Double.parseDouble(split[4]);
+            employees[i] = new Employee(firstName, lastName, pesel, department, salary);
+        }
+        return employees;
+    }
+
+    private static int countLines(File file) throws FileNotFoundException {
+        Scanner scanner = new Scanner(file);
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY